README.md

```diff
@@ -56,6 +56,8 @@ Confirm that it was installed correctly: `python3 --version`. You should see `Py
 
 ##### Install pip and virtualenv
 
-pip is the preferred way to install dependencies, including Django.  Documentation for installing pip can be found [here](https://pip.pypa.io/en/stable/installing/).  Once pip is installed, it is trivial to install the virtualenv package (for isolating project dependencies).  In the Terminal type `pip install virtualenv`.  
+pip is the preferred way to install dependencies, including Django.  Documentation for installing pip can be found [here](https://pip.pypa.io/en/stable/installing/).  Once pip is installed, it is trivial to install the virtualenv package (for isolating project dependencies).  If your Python version is creater than 3.3, then `virtualenv` comes installed along with by default.  Otherwise, in the Terminal type `pip install virtualenv`.  
 
-Once virtualenv has been installed, navigate to the cloned project directory and create a local virtualenv folder with `virtualenv venv`. This will create a folder in you project directory called `venv/` which will store your local project dependencies.  Activate you virtual environment by typing `source venv/bin/activate`, and install your local dependencies in the virtual environment by typing `pip install -r requirements.txt`, which will download and install all the project required software listed in the `requirements.txt` file that was cloned with the project. The virtual environment can be turned off anytime by typing `deactivate` in the Terminal in the project base directory.
+Once virtualenv has been installed, navigate to the cloned project directory and create a local virtualenv folder with `virtualenv venv`.  If virtualenv is included in your Python installation, then the command is `python3 -m venv venv`. This will create a folder in you project directory called `venv/` which will store your local project dependencies.  Activate you virtual environment by typing `source venv/bin/activate`, and install your local dependencies in the virtual environment by typing `pip install -r requirements.txt`, which will download and install all the project required software listed in the `requirements.txt` file that was cloned with the project. The virtual environment can be turned off anytime by typing `deactivate` in the Terminal in the project base directory.
+
+On some versions of MacOS, the dependencies installation process may be interrupted by a failure to install the dependency `psycoph`. If that happens and dependency installation is aborted, run `pip install psycopg2==2.7.5`, and then run `pip install -r requirements.txt` again.
```

Comment:
Nit: typo?
psycoph -> psycopg

Reply:
oops, good catch. psycopg is the correct spelling